var/spack/repos/builtin/packages/kokkos-kernels-legacy/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class KokkosKernelsLegacy(MakefilePackage):
+    """Kokkos C++ Performance Portability Programming EcoSystem: Math Kernels -
+    Provides BLAS, Sparse BLAS and Graph Kernels."""
+
+    homepage = "https://github.com/kokkos/kokkos-kernels"
+    url      = "https://github.com/kokkos/kokkos-kernels/archive/2.7.00.tar.gz"
+
+    version('2.7.00', sha256='adf4af44eadbdfbeb9ec69dd5fae4e2852bd1fbe4a69213efd199e49f4098254')
+    version('2.6.00', sha256='14ebf806f66b9ca73949a478b8d959be7fa1165a640935760a724d7cc0a66335')
+    version('2.5.00', sha256='2c2289da3a41dafd97726e90507debafbb9f5e49ca5b0f5c8d1e044a5796f000')
+    version('develop', git='https://github.com/kokkos/kokkos-kernels',
+            branch='develop')
+
+    # make sure kokkos kernels version matches kokkos
+    depends_on('kokkos@2.5.00', when='@2.5.00')
```

Comment:
Could you re-order these `depends_on` to match the order of the versions above?

Reply:
I'd prefer to not modify the legacy package any more than just moving it?

Reply:
Yeah, let's not touch it.